VSRAD.Package/Server/ActionRunner.cs

```diff
@@ -254,6 +270,13 @@ private async Task<StepResult> DoCopySingleFileAsync(CopyFileStep step)
             return new StepResult(true, "", "");
         }
 
+        private async Task<StepResult> DoOpenInEditorAsync(OpenInEditorStep step)
```

Comment:
This method was moved to `IActionRunController.OpenFileInVsEditorAsync` and can be removed from `ActionRunner`.